scraper/src/config/browser_handler.py

```diff
@@ -32,7 +33,7 @@ def init(config_original_content, js_render, user_agent):
             if not os.path.isfile(CHROMEDRIVER_PATH):
                 raise builtins.Exception(
                     f"Env CHROMEDRIVER_PATH='{CHROMEDRIVER_PATH}' is not a path to a file")
-            driver = webdriver.Chrome(
+            driver = webdriver.Chrome(  # pylint: disable=redundant-keyword-arg
```

Comment:
Look at my editor again I think I must have had a cached version or something. All the tests pass here, but they either shouldn't be or there are some tests missing. `Chrome` looks like:

```py
class Chrome(options: Options = None, service: Service = None, keep_alive: bool = True):
    ...
```

So `options` are doubled and the first is sent as a path which shouldn't work.

Reply:
I found the problem and there must be tests missing because this shouldn't work. `executable_path` was depreciated and has been removed in this version. I checking to see if there is a new way to specify it.

https://github.com/SeleniumHQ/selenium/pull/12030/files

Reply:
According to what I have found this argument isn't present anymore because Selenium downloads the driver and manages it itself now. With that the `CHROMEDRIVER_PATH` should be removed all together. https://www.selenium.dev/documentation/webdriver/troubleshooting/errors/driver_location/#use-the-latest-version-of-selenium

@alallema what are your thoughts on this? The part I'm not comfortable with is there doesn't seem to be a test for this so we can remove it, but I'm not sure we can be comfortable it works. I could try to think of a test we could add for it.

Reply:
> According to what I have found this argument isn't present anymore because Selenium downloads the driver and manages it itself now. With that the `CHROMEDRIVER_PATH` should be removed all together. https://www.selenium.dev/documentation/webdriver/troubleshooting/errors/driver_location/#use-the-latest-version-of-selenium
> 
> @alallema what are your thoughts on this? The part I'm not comfortable with is there doesn't seem to be a test for this so we can remove it, but I'm not sure we can be comfortable it works. I could try to think of a test we could add for it.

I'm thinking the same thing, I've had a look at the documentation and the links you've provided, and in my opinion, we can remove all the line referring to `CHROMEDRIVER_PATH` and just keep the `options`.
I've tested it myself on the PR, and everything seems to be working fine.

About the tests, I agree so much there are very few tests in this package it's a problem. If you have an idea for testing this, I'd love to hear it, but it's possible that everything has to be created on this side. So if it's too long, I can create an issue about it and add the tests later.

Thanks again @sanders41 for your research on this topic, it's really helpful

Reply:
I removed the reference to `CHROMEDRIVER_PATH`.

For the tests, in theory if the documentation is correct, this should allow us to fix #177 which would have also caught this issue if these tests were on. Turning those tests back on would be a good start and give a better idea where we are overall. Since that is outside the scope of this PRs intent it probably makes sense to do it in a separate PR.